src/quo2/components/drawers/drawer_top/view.cljs

```diff
@@ -0,0 +1,157 @@
+(ns quo2.components.drawers.drawer-top.view
+  (:require [quo2.theme :as quo.theme]
+            [quo2.components.markdown.text :as text]
+            [quo2.components.drawers.drawer-top.style :as style]
+            [quo2.components.buttons.button.view :as button]
+            [quo2.components.tags.context-tag.view :as context-tag]
+            [react-native.core :as rn]
+            [quo2.components.icon :as icons]
+            [quo2.components.avatars.account-avatar.view :as account-avatar]
+            [quo2.components.avatars.icon-avatar :as icon-avatar]
+            [quo2.components.avatars.user-avatar.view :as user-avatar]
+            [quo2.foundations.colors :as colors]
+            [utils.i18n :as i18n]))
+
+(defn- left-image
+  [{:keys [type account-avatar-customization-color account-avatar-emoji icon-avatar profile-picture]}]
+  (case type
+    :account         [account-avatar/view
+                      {:customization-color account-avatar-customization-color
+                       :size                32
+                       :emoji               account-avatar-emoji
+                       :type                :default}]
+    :keypair         [icon-avatar/icon-avatar
+                      {:size    :medium
+                       :icon    icon-avatar
+                       :border? true
+                       :color   :neutral}]
+
+    :default-keypair [user-avatar/user-avatar
+                      {:size                  :small
+                       :identification-right? true
+                       :nft-avatar?           false
+                       :profile-picture       profile-picture}]
+    nil))
+
+(defn- subtitle
+  [{:keys [type theme blur? keycard? networks description community-name community-logo]}]
+  (cond
+    (= :keypair type)                    [rn/view {:style style/row}
```

Comment:
Usually in our repo we see line breaks after the conditions to avoid right-aligning too much, otherwise it forces zprint to add line breaks too often (because its configured line length is 105 characters).

Usually when the `cond` body extends to a few dozen lines, it's common to add an empty line between sections of the `cond` macro. Side note: It's also possible to configure zprint to add line breaks between pairs in the cond macro (https://github.com/kkinnear/zprint/blob/main/doc/reference.md#map-nl-pair-nl-binding-nl).

So my suggestion here would be to style like this for improved readability:

```clojure
(defn- subtitle
  [{:keys [type theme blur? keycard? networks description community-name community-logo]}]
  (cond
    (= :keypair type)
    [rn/view {:style style/row}
     [text/text
      {:size   :paragraph-2
       :weight :regular
       :style  (style/description theme blur?)}
      (if keycard?
        (i18n/label :t/on-keycard)
        (i18n/label :t/on-device))]
     (when keycard?
       [icons/icon :i/keycard-card
        {:color (colors/theme-colors colors/neutral-50 colors/neutral-40 theme)}])]

    (= :account type)
    [rn/view {:style style/row}
     (for [network networks]
       ^{:key (str network)}
       [network-view network])
     [text/text
      {:size   :paragraph-2
       :weight :regular
       :style  (style/description theme blur?)}
      description]]

    (= :default-keypair type)
    [text/text
     {:accessibility-label :default-keypair-text
      :size                :paragraph-2
      :weight              :regular
      :style               (style/description theme blur?)}
     (str description " ∙ " (i18n/label :t/on-device))]

    (= :context-tag type)
    [rn/view
     {:accessibility-label :context-tag-wrapper
      :style               {:flex-wrap :wrap}}
     [context-tag/view
      {:type           :community
       :community-name community-name
       :community-logo community-logo
       :size           24}]]

    (and (not= :label type) description)
    [text/text
     {:size   :paragraph-1
      :weight :regular
      :style  (style/description theme blur?)}
     description]))
```

Reply:
Done. Very good point 👍